File: CPP/Baju.cpp
#include <bits/stdc++.h>
#include "Aksesoris.cpp"
using namespace std;

class Baju : public Aksesoris {
    private:
        string untuk;
        string size;
        string merk;
    
    public:

        Baju(string id, string nama, int stok, double harga, string jenis, string bahan, string warna, 
             string untuk, string size, string merk) 
            : Aksesoris(id, nama, stok, harga, jenis, bahan, warna) { // Panggil constructor parent
            this->untuk = untuk;
            this->size = size;
            this->merk = merk;
        }

        // Getter dan Setter
        string getUntuk() const { return untuk; }
        void setUntuk(string untuk) { this->untuk = untuk; }
    
        string getSize() const { return size; }
        void setSize(string size) { this->size = size; }
    
        string getMerk() const { return merk; }
        void setMerk(string merk) { this->merk = merk; }
};

File: PHP/Baju.php
<?php

// Include Aksesoris.php untuk mewarisi kelas Aksesoris
include 'Aksesoris.php';

class Baju extends Aksesoris {
    private $untuk = '';
    private $size = '';
    private $merk = '';

    // constructor
    public function __construct($id = '', $nama = '', $stok = 0, $harga = 0.0, $foto = '', $jenis = '', $bahan = '', $warna = '', $untuk = '', $size = '', $merk = ''){
        // Memanggil constructor parent (Aksesoris)
        parent::__construct($id, $nama, $stok, $harga, $foto, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    // setter and getter for untuk
    public function setUntuk($untuk){
        $this->untuk = $untuk;
    }

    public function getUntuk(){
        return $this->untuk;
    }

    // setter and getter for size
    public function setSize($size){
        $this->size = $size;
    }

    public function getSize(){
        return $this->size;
    }

    // setter and getter for merk
    public function setMerk($merk){
        $this->merk = $merk;
    }

    public function getMerk(){
        return $this->merk;
    }
}

?>
